src/effects/effectchainslot.h

```diff
@@ -133,11 +135,24 @@ class EffectChainSlot : public QObject {
     ControlPushButton* m_pControlChainEnabled;
     ControlObject* m_pControlChainMix;
     ControlObject* m_pControlChainSuperParameter;
+    ControlObject* m_pControlChainSuperForceUpdate;
     ControlPushButton* m_pControlChainInsertionType;
     ControlObject* m_pControlChainSelector;
     ControlPushButton* m_pControlChainNextPreset;
     ControlPushButton* m_pControlChainPrevPreset;
 
+    /**
```

Comment:
For the sake of this working for 2.1 and an impending feature freeze, I think this should wait. It is working reliably as it is. Catching the corner cases of a more complicated change like that could be tricky.